memcached-operator/deploy/olm-catalog/memcached-operator/memcached-operator.package.yaml

```diff
@@ -1,5 +1,5 @@
 channels:
-- currentCSV: memcached-operator.v0.10.0
```

Comment:
We should be adding each working CSV name to `channels` with name `spec.version` from the CSV. The default channel should be updated to the newest channel name too.

Reply:
Could you give an example over how to fix it? Also, could we not do it in another PR? It shows not part of the scope to upgrade the SDK version on it.

Reply:
Tks I catcher now 👍

Reply:
Sorry just for the sake of my own understanding, what was the resolution to this?

Reply:
@estroz Can you clarify your original suggestion. Are you thinking we can have multiple channels, like this:

```yaml
channels:
- currentCSV: memcached-operator.v0.11.0
  name: v0.11.x
- currentCSV: memcached-operator.v0.10.0
  name: v0.10.x
defaultChannel: v0.11.x
```

Reply:
I spoke with @estroz and he explained that the name should be the version `0.11.0`. 

@estroz is it ok for you now?

Reply:
The version format in the channel name doesn't matter to me. @joelanford the current change looks good?

Reply:
As is, I don't think I agree with this change. I think we either:
1. Stick with `alpha` and don't add any more _new_ channels.
2. OR Use `v0.11.x`, `v0.10.x`, etc. for each CSV we have that goes with a specific version of the SDK. That way if we make patch releases, we don't have to have a whole new channel for the CSV. 

To be totally honest, I think it's a little weird to have CSVs per SDK version, since that isn't how a "real" operator would do things.

My vote would be to go with option 1 since that seems to be the way we've been doing it.

Reply:
+1 for @joelanford, @estroz are you ok with? Can I do the option 1 here?

Reply:
@joelanford what you're implying is that we should update the `alpha` channel with the new CSV version on each release?

An issue crops up here with the CSV's `spec.replaces`. If OLM can't resolve that CSV, then installation will fail. So each `spec.replaces` must reference a CSV name specified in the package manifest. That is why I suggested we use versioned channel names. We may have to remove `spec.replaces` from all CSV versions if we do not want to also expand the number of channels.

Reply:
Since the goal here is just to migrate to upper version, I think that we should keep it as it has been done so far and then we discuss these points to get done in another PR. Tracked https://github.com/operator-framework/operator-sdk-samples/issues/89 for it. WDYT? Could we move forward in this way? 

@joelanford @estroz